packages/opentelemetry-exporter-prometheus/test/PrometheusSerializer.test.ts

```diff
@@ -302,7 +302,7 @@ describe('PrometheusSerializer', () => {
     describe('with HistogramAggregator', () => {
       mockAggregator(HistogramAggregator);
 
-      it('serialize metric record with MinMaxLastSumCountAggregator aggregator', async () => {
+      it('serialize metric record with MinMaxLastSumCountAggregator aggregator, cumulative', async () => {
```

Comment:
```suggestion
      it('serialize metric record with HistogramAggregator aggregator, cumulative', async () => {
```

I believe this case name is a typo previously.

Reply:
fixed